easyrest/scripts/initialize_db.py

```diff
@@ -5,15 +5,28 @@
 from sqlalchemy.exc import OperationalError
```

Comment:
Please fix this module according to my comments in #3:
- docstrings
- `print` as statements
- make messages more informative